### DISCUSSION TITLE
How can I do an out-of-band update on the tree-sitter library used by this plugin?

### DISCUSSION THREAD

**Discussion by ahelwer** [nvim-treesitter/nvim-treesitter#1860] [Q&A] [answered]
I'll be making some changes to tree-sitter's C library for this issue: https://github.com/tree-sitter/tree-sitter/issues/1405

I would like to preview these changes inside neovim. How do I replace this plugin's tree-sitter library with a locally-built fork?

**Comment by stsewd** [comment, marked as answer]
You'll need to compile neovim with that version, see for examples this PR where ts is updated https://github.com/neovim/neovim/pull/14188/files

**Reply by ahelwer** [reply]
Thanks, I successfully build a neovim release with the tree-sitter fork! https://github.com/tlaplus-community/neovim/releases/tag/v0.5.2
